src/eclingo/parsing/transformers/parser_negations.py

```diff
@@ -25,21 +28,28 @@ class SimplifyStrongNegationsTransformer(Transformer):
 
 
 class StrongNegationToAuxiliarTransformer(Transformer):
-    def __init__(self, strong_negation_prefix="sn"):
+    def __init__(self, use_reification, strong_negation_prefix="sn"):
         self.strong_negation_prefix = strong_negation_prefix
         self.replacement = set()
+        self.reification = use_reification
 
     def visit_UnaryOperation(self, x):
         assert x.operator_type == ast.UnaryOperator.Minus
         assert x.argument.ast_type == ast.ASTType.Function
         x = simplify_strong_negations(x)
         name = x.argument.name
+
         location = x.argument.location
-        aux_name = self.strong_negation_prefix + "_" + name
         arguments = x.argument.arguments
         external = x.argument.external
+
+        if self.reification:
+            aux_name = "-" + name
```

Comment:
This looks strange. 
Why are you using this as a name?
Test seem to be correct, but I expected to use ast.UnaryOperator.Minus.